services/otel-gateway/generate.py

```diff
@@ -0,0 +1,31 @@
+from opentelemetry import metrics, trace
+from opentelemetry.exporter.otlp.proto.http.trace_exporter import \
+    OTLPSpanExporter
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import SimpleSpanProcessor
+
+# set up trace exporting
+provider = TracerProvider()
+provider.add_span_processor(SimpleSpanProcessor(OTLPSpanExporter()))
+trace.set_tracer_provider(provider)
+tracer = trace.get_tracer("testscope")
+
+
+def generate_trace():
+    with tracer.start_as_current_span("testname") as span:
```

Comment:
Why are test values used here? What have I missed?

Reply:
This is the test traces that we use in tests.py to verify the gateway is working, and these values are asserted against there.

They are arbitrary, and in CI/local dev it never reaches honeycomb, as it's received by the test collector that is pretending to be honeycomb in the test suite.

The reason I generated the trace in a separate file is so we can easily re-use manually  test against actual honeycomb, using a development environment API key, as a full integration test.

Reply:
Ah, I see. I didn't realise it was for just for the tests, could it be renamed to make it clearer? It's almost a fixtures file, I suppose.

Reply:
I've merged `generate.py` into `tests.py`, so this is more obvious.

I added an `if __name__ == '__main__':` section to tests.py, so it can be used in `test-integration` the same way `generate.py` was.